src/pages-and-resources/discussions/data/api.js

```diff
@@ -11,6 +11,9 @@ function normalizeLtiConfig(data) {
     consumerKey: data.lti_1p1_client_key,
     consumerSecret: data.lti_1p1_client_secret,
     launchUrl: data.lti_1p1_launch_url,
+    piiSharing: 'pii_share_username' in data || 'pii_share_email' in data,
```

Comment:
If the either of these toggles are off, will that toggle not be included in the api response?
e.g. if a course sets `pii_share_username` to false, will that key be excluded from api response?

Reply:
No, there are two flag levels here:
1. You have to set a course-level flag to enable showing these options. If that flag is not set then these keys will be absent. 
2. If that flag is set, then by default these values will be false.

So if the keys are absent, we treat that as this course not being configured/enabled for PII sharing.